backend/routers/tickets.py

```
@@ -0,0 +1,48 @@
+from fastapi import APIRouter, HTTPException, Depends
+from backend.database import supabase
+from backend.schemas import *
+
+router = APIRouter(prefix="/tickets", tags=["Tickets"])
+api_router = APIRouter(prefix="/api", tags=["API Decoupled"])
+
+@router.get("/")
+async def get_tickets():
+    if not supabase: return []
```

Ruff (0.15.15)

[error] Multiple statements on one line (colon)

(E701)